rust/protocol/src/curve.rs

```diff
@@ -299,13 +315,15 @@ impl KeyPair {
         }
     }
 
+    /// Instantiate an identity from a known public/private key pair.
     pub fn new(public_key: PublicKey, private_key: PrivateKey) -> Self {
         Self {
             public_key,
             private_key,
         }
     }
 
+    /// Instantiate an identity from byte strings for public and private keys.
```

Comment:
I'd specifically use the word "serialized" here rather than just "byte strings", since public keys are "serialized" with the leading "type" byte and that's what's expected here.

Reply:
Yes! I learned that the hard way by messing around last night! I'll make note of this.